install/install_openadapt.sh

```diff
@@ -115,10 +114,36 @@ if ! CheckCMDExists "git"; then
     BrewInstall "git"
 fi
 
+# for scrubbing
 if ! CheckCMDExists "tesseract"; then
     BrewInstall "tesseract"
 fi
 
+# for SVG mixin
+if ! CheckCMDExists "vtracer"; then
+    if ! CheckCMDExists "cargo"; then
+       brew install rust
+       if ! CheckCMDExists "cargo"; then
+            echo "Failed to download rust"
+            Cleanup
+            exit 1
+        fi
+    fi
+    RunAndCheck "cargo install vtracer" "install vtracer"
```

Comment:
I ran `cargo install vtracer` and got:

```
warning: be sure to add `/Users/abrichr/.cargo/bin` to your PATH to be able to run the installed binaries
```

Then created a new terminal:

```
% vtracer
zsh: command not found: vtracer
```

So it seems that modifying the path is necessary for this to work.